pkg/pillar/cas/containerd.go

```diff
@@ -371,7 +371,8 @@ func (c *containerdCAS) IngestBlob(ctx context.Context, blobs ...types.BlobStatu
 			}
 			i := len(m.Layers)
 			info.Labels[fmt.Sprintf("%s.%d", containerdGCRef, i)] = m.Config.Digest.String()
-
+			// set eve-downloaded label for this blob
+			info.Labels["eve-downloaded"] = "true"
```

Comment:
This `eve-downloaded` and `true` should be moved to `const` and referenced

Reply:
Just this one still has it here, rather than referencing a constant.

Reply:
Wow how is it possible when I did git squash of the two commits !! I will fix that